content/en/blog/creating-blog-with-nuxt-content.md

```diff
@@ -288,8 +288,6 @@ All other tags that come from our YAML front matter can be styled as normal eith
 
 To use scoped styles with the nuxt-content class you need to use a deep selector: `/deep/`, `::v-deep` or `>>>`
 
-All other tags that come from our YAML front matter can be styled as normal either using [TailwindCSS](https://tailwindcss.com/) or adding css in the style tag.
```

Comment:
This is actually correct and needs to remain in here. You can style anything that you add in your YAML just like you would normal html. 
content/hello.md
```
---
title: my title
---
```
pages/index.vue
` <h1 class="title">{{ page.title }} </h1>`

Reply:
Thank you for your reply. Actually I removed it because it is a duplicated paragraph. Isn’t it? Sorry I missunderstand it if it isn’t.

Reply:
ahhh ok. I see. It got added again somehow. Ok perfect. Always good to explain why you are removing something as it is not always visible in the pr diff especially when the duplicated content is not directly above/below each other. I suggest in future using the description box to explain what you are removing/editing/amending things.

Thanks for your contributions